.github/workflows/_hugo.yml

```
@@ -67,16 +41,16 @@ jobs:
+
       - name: Upload artifact
         uses: actions/upload-pages-artifact@v3
         with:
           path: ./public
 
-  # Deployment job
   deploy:
     environment:
       name: github-pages
-      url: ${{ steps.deployment.outputs.page_url }}
+      url: ${{ steps.build.outputs.page_url }}
```

actionlint

property "build" is not defined in object type {deployment: {conclusion: string; outcome: string; outputs: {page_url: string}}}

(expression)